Dockerfile

```diff
@@ -0,0 +1,16 @@
+# A simple container for variant-service.
+# Runs service on port 80.
+# Healthchecks service up every 5m.  
+
+FROM python:3.10
+RUN apt update ; apt install -y rsync
+RUN pip install pipenv uvicorn[standard]
+COPY . /app
+WORKDIR /app
+RUN if [ ! -f "Pipfile.lock" ] ; then pipenv lock ; else echo Pipfile.lock exists ; fi
+RUN pipenv sync
```

Comment:
Instead of pipenv, can we just install deps using pip?

Reply:
Can be done , but this would require change in Docker commands. I have followed the existing setup of Variation Normalizer mentioned in Readme which is using pip. Any particular reason for moving to pip approach.

Reply:
@rajatkapoordfci We've been removing `pipenv` from our projects in favor of `pip` (and we'll probably switch to [uv](https://github.com/astral-sh/uv)). pipenv is also pretty slow